packages/expo-doctor/__mocks__/fs.js

```diff
@@ -1,2 +1,3 @@
-import { fs } from 'memfs';
+import { fs, realpathSync } from 'memfs';
+module.exports = realpathSync;
 module.exports = fs;
```

Comment:
Does multiple `module.exports = ...` work? I think the `realPathSync` is just getting overwritten when exporting, right?